webapp/scripts/gen_lang_imports.mjs

```diff
@@ -37,11 +37,16 @@ fs.readdirSync('./channels/src/i18n').forEach(file => {
   }
 });
 
+lines += `
+type TranslationsMap = {
```

Comment:
This is not JS Map type per se but 0/5

Reply:
I'm not as well versed here: is there a better way, @M-ZubairAhmed?

Reply:
Sorry This is good as is. i pointed out that we should not call TranslationsMap as Map as its a Record rather than a Map